js/Events/newEvents.js

```diff
@@ -56,16 +54,14 @@ interviewApp.Events.EventsNew = function () {
         interviewApp.Modules.EvaluationsModule.init();
     };
 
-    const isCandidateEmpty = function() {
-        const candidate = document.getElementsByName('candidate')[0]; 
+    const isCandidateEmpty = function () {
```

Comment:
Judging by the name of this method, I'd expect it to always return a boolean value, but it looks like it sometimes returns boolean, while other times it returns DOM elements. There's always a case to be made about consistency in your code, and more so with respect to input / output. If you want your APIs to be clear, concise, fluent and predictable, one of the things that you'll want to pay special attention to is data types, especially for inputs and outputs.

Reply:
I will consider that.